src/components/cylc/tree/index.js

```diff
@@ -155,18 +153,64 @@ function createJobDetailsNode (job) {
     },
     {
       title: 'latest message',
-      property: job.latestMessage
+      value: job.latestMessage
```

Comment:
Going away in #596, but included it to make sure I wasn't mutating any object incorrectly (i.e. maybe somewhere else I could be doing `obj.value=''` and then it would have `title`, `property`, and `value`. Will rebase and remove it if #596 is merged first.